set.mm

```diff
@@ -644104,182 +644101,206 @@ base set if and only if the neighborhoods (convergents) of every point
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
-  $c MinUni $.
-
-  $( Extend class notation with the class of minimal universes. $)
-  cmnu $a class MinUni $.
-
   ${
-    $d y z f w v i u $.
-    $( Define the class of minimal universes.  These are the sets which satisfy
-       the predicate on ` y ` in ~ rr-grothprim , except for the ` x e. y `
-       clause.
+    $d z w v U f i k m n q p l $.  $d z w u U f i k m n r p l $.
+    ismnu.1 $e |- M = { k | A. l e. k A. m ( ~P l C_ k /\ E. n e. k
+      ( ~P l C_ n /\ A. p e. l ( E. q e. k ( p e. q /\ q e. m )
+        -> E. r e. m ( p e. r /\ U. r C_ n ) ) ) ) } $.
+    $( Properties of a minimal universe.  These are the sets which satisfy the
```

Comment:
If my understanding is correct, then I think it would be clearer to replace the beginning of the comment with:

The hypothesis of this theorem defines a class ` M ` of sets that we temporarily call "minimal universes", and which will turn out in ~ grumnueq to be exactly Grothendicek universes. Minimal universes are sets that satisfy...